edu.stanford.protege.json

```
@@ -0,0 +1,74 @@
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk8"],
+    "command": "protege",
+    "finish-args": [
+        "--env=PATH=/app/bin:/app/jre/bin:/usr/bin",
+        "--socket=x11",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": ["/usr/lib/sdk/openjdk8/install.sh"]
+        },
+        {
+            "name": "protege",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/protegeproject/protege-distribution/releases/download/v5.5.0/Protege-5.5.0-platform-independent.zip",
+                    "sha512": "7dac94f2cc9f38c0993ab374abdc70d6dd581f3929b6f5d19e2a19b6df4ffc4154db697964b055451070cf1afc1c6f0bb2ed741751c3a6517f6ea6b050f70a55"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege-16.png"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege-32.png"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege-48.png"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege-64.png"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege-128.png"
+                }
+            ],
+            "build-commands": [
+                "mkdir -p /app/{bin,share}",
+                "cp -R . /app/share/protege",
+                "chmod +x /app/share/protege/run.sh",
+                "ln -s /app/share/protege/run.sh /app/bin/protege",
+                "install -Dm644 edu.stanford.protege.desktop -t /app/share/applications",
+                "install -Dm644 edu.stanford.protege.appdata.xml -t /app/share/metainfo",
+                "install -Dm644 edu.stanford.protege-16.png /app/share/icons/hicolor/16x16/edu.stanford.protege.png",
+                "install -Dm644 edu.stanford.protege-32.png /app/share/icons/hicolor/32x32/edu.stanford.protege.png",
+                "install -Dm644 edu.stanford.protege-48.png /app/share/icons/hicolor/48x48/edu.stanford.protege.png",
+                "install -Dm644 edu.stanford.protege-64.png /app/share/icons/hicolor/64x64/edu.stanford.protege.png",
+                "install -Dm644 edu.stanford.protege-128.png /app/share/icons/hicolor/128x128/edu.stanford.protege.png"
```

```suggestion
                "install -Dm644 edu.stanford.protege-16.png /app/share/icons/hicolor/16x16/apps/edu.stanford.protege.png",
                "install -Dm644 edu.stanford.protege-32.png /app/share/icons/hicolor/32x32/apps/edu.stanford.protege.png",
                "install -Dm644 edu.stanford.protege-48.png /app/share/icons/hicolor/48x48/apps/edu.stanford.protege.png",
                "install -Dm644 edu.stanford.protege-64.png /app/share/icons/hicolor/64x64/apps/edu.stanford.protege.png",
                "install -Dm644 edu.stanford.protege-128.png /app/share/icons/hicolor/128x128/apps/edu.stanford.protege.png"
```

This probably is why the build fail.